src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDriver.java

```diff
@@ -292,18 +293,19 @@ public String toString() {
 
 enum SQLServerDriverBooleanProperty
 {
-	DISABLE_STATEMENT_POOLING          ("disableStatementPooling",         true),
-	ENCRYPT                            ("encrypt",                         false),	
-	INTEGRATED_SECURITY                ("integratedSecurity",              false),
-	LAST_UPDATE_COUNT                  ("lastUpdateCount",                 true),
-	MULTI_SUBNET_FAILOVER              ("multiSubnetFailover",             false),
-	SERVER_NAME_AS_ACE                 ("serverNameAsACE",                 false),
-	SEND_STRING_PARAMETERS_AS_UNICODE  ("sendStringParametersAsUnicode",   true),
-	SEND_TIME_AS_DATETIME              ("sendTimeAsDatetime",              true),
-	TRANSPARENT_NETWORK_IP_RESOLUTION  ("TransparentNetworkIPResolution",  true),
-	TRUST_SERVER_CERTIFICATE           ("trustServerCertificate",          false),
-	XOPEN_STATES                       ("xopenStates",                     false),
-	FIPS                               ("fips",                            false);
+	DISABLE_STATEMENT_POOLING                 ("disableStatementPooling",                   true),
+	ENCRYPT                                   ("encrypt",                                   false),	
+	INTEGRATED_SECURITY                       ("integratedSecurity",                        false),
+	LAST_UPDATE_COUNT                         ("lastUpdateCount",                           true),
+	MULTI_SUBNET_FAILOVER                     ("multiSubnetFailover",                       false),
+	SERVER_NAME_AS_ACE                        ("serverNameAsACE",                           false),
+	SEND_STRING_PARAMETERS_AS_UNICODE         ("sendStringParametersAsUnicode",             true),
+	SEND_TIME_AS_DATETIME                     ("sendTimeAsDatetime",                        true),
+	TRANSPARENT_NETWORK_IP_RESOLUTION         ("TransparentNetworkIPResolution",            true),
+	TRUST_SERVER_CERTIFICATE                  ("trustServerCertificate",                    false),
+	XOPEN_STATES                              ("xopenStates",                               false),
+    FIPS                                      ("fips",                                      false),
```

Comment:
Agreed. problem was I introduced a longer name than was there before so I wanted to keep things according to the file standard.